common/lifecyleMgr.go

```diff
@@ -128,6 +131,9 @@ func (lcm *lifecycleMgr) Progress(o OutputBuilder) {
 }
 
 func (lcm *lifecycleMgr) Info(msg string) {
+
+	msg = lcm.logSanitizer.SanitizeLogMessage(msg) // sometimes error-like text comes through Info, before the final "we've failed, please stop now" signal comes to Error. So we sanitize in both places.
```

Comment:
When do we output errors as INFO? Example?

Reply:
In the case I tested it on!  Which was the enumerator being unable to connect. (I had to turn the retry count down to make the test run in reasonable time).  There were multiple lines of output describing the error.  Basically a bunch of stuff was logged with Info, and then the actual error was logged.  Basically is if Info was being used as lcm.Println, and Error was being used as lcm.PrintlnAndTerminate.

Reply:
Ok! Sounds good